api_gateway/src/main/kotlin/org/thechance/api_gateway/endpoints/UserRoute.kt

```diff
@@ -64,6 +66,45 @@ fun Route.userRoutes() {
                 respondWithResult(HttpStatusCode.OK, users)
             }
         }
+
+        authenticateWithRole(Role.END_USER) {
```

Comment:
I tried to add to favorite from admin account and gives me access denied
it supposed that i have permission for end user too